control/controldisplay/summary_status_row.go

```diff
@@ -0,0 +1,68 @@
+package controldisplay
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/turbot/go-kit/helpers"
+	"github.com/turbot/steampipe/control/controlexecute"
+	"golang.org/x/text/language"
+	"golang.org/x/text/message"
+)
+
+type SummaryStatusRowRenderer struct {
+	resultTree *controlexecute.ExecutionTree
+	width      int
+	status     string
+}
+
+func NewSummaryStatusRowRenderer(resultTree *controlexecute.ExecutionTree, width int, status string) *SummaryStatusRowRenderer {
+	return &SummaryStatusRowRenderer{
+		resultTree: resultTree,
+		width:      width,
+		status:     status,
+	}
+}
+
+func (r *SummaryStatusRowRenderer) Render() string {
+
+	colorFunction := ControlColors.StatusColors[strings.ToLower(r.status)]
```

Comment:
do we need ToLower?

If so - shouldn't we do it in the switch statement as well?

Reply:
do we have consts defined for status - if not should we?

Reply:
Do we need the ToLower because we are not sure what the control will output? Are we case insensitive elsewhere as well?

if we need to do ToLower twice, let's pull status into a variable